about.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <div class="ham">
+        <span class="bar1"></span>
+        <span class="bar2"></span>
+        <span class="bar3"></span>
+      </div>
+      <div class="user-mod-container">
+        <ul class="user-mod">
+          <li>
+            <a href="#" class="twitter"
+              ><img
+                src="./css/images/Icons/twitter-logo.png"
+                alt="Twitter account"
+            /></a>
+          </li>
+          <li>
+            <a href="#" class="fb"
+              ><img
+                src="./css/images/Icons/facebook-logo.png"
+                alt="Facebook account"
+            /></a>
+          </li>
+          <li>English</li>
+          <li>My Page</li>
+          <li class="logout">Logout</li>
+        </ul>
+        <nav>
+          <ul class="navbar-hide">
+            <li>
+              <a href="./index.html"
+                ><img src="./css/images/logo/The-Legends-logo.png" alt="logo"
+              /></a>
+            </li>
+            <li><a href="./index.html">Home</a></li>
+            <li><a href="#1"> Program</a></li>
+            <li><a href="#2">Join</a></li>
+            <li><a href="#3">Sponser</a></li>
+            <li><a href="#4">News</a></li>
+            <li class="campaign"><a href="#5">CC Campaign</a></li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="about-main-container">
+      <div>
+        <ul class="mini-nav">
+          <li class="mini-nav-items">
+            <a href="./index.html"
+              ><img src="./css/images/Icons/home.png" alt="home"
+            /></a>
+          </li>
+          <li class="mini-nav-items">></li>
+          <li class="mini-nav-items"><a href="./about.html">About</a></li>
+        </ul>
+      </div>
+      <div class="about-main">
```

Comment:
- [ ] **Required:** Change your div tags to more semantic tags for better SEO listings.

![image](https://user-images.githubusercontent.com/27709832/163438806-d6a7899b-5b0e-4dcc-ba4f-b8a577c60f4d.png)